src/app/archive/create_schema.sql

```diff
@@ -5,6 +5,19 @@ CREATE TABLE public_keys
 
 CREATE INDEX idx_public_keys_value ON public_keys(value);
 
+CREATE TABLE timing_info
+( id                      serial    PRIMARY KEY
+, public_key_id           int       NOT NULL REFERENCES public_keys(id)
```

Comment:
do timings only apply to accounts using the default token?

if not, the public key does not identify the account without a token